### ISSUE TITLE
image save issue on Windows

### ISSUE THREAD

**Issue by deufrai** [gehelem/als#195]
Running als v0.7-beta2 on Windows 10

```
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.065581 ***************************************************************************
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.065581 User config file dump - START
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.065581 scan_folder_path               : C:/als_scanz
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.065581 work_folder_path               : C:/als_work
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.065581 www_dedicated_folder           : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 web_folder_path                : C:/als_work
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 alignment_minimum_match_count  : 22
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 use_master_dark                : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 use_hot_pixel_remover          : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 web_refresh_period             : 5
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 log_level                      : DEBUG
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 lang                           : sys
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 full_screen                    : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 window_geometry                : 0,23,1920,1017
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.066633 night_mode                     : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.067673 save_on_stop                   : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.067673 master_dark_file_path          : 
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.067673 profile                        : 0
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.067673 www_server_port                : 8000
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.067673 User config file dump - END
=MainThread   als.config           DEBUG    2024-10-07 18:49:36.067673 ***************************************************************************
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.067673 ***************************************************************************
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.067673 System info dump - START
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.067673 Platform name         : win32
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.067673 Platform architecture : ('64bit', 'WindowsPE')
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 Machine name          : AMD64
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 CPU type              : Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 CPU count             : 4
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 OS name               : Windows
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 OS release            : 10
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 Available memory      : 4.446 GiB
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.068722 Python version        : 3.7.7 (tags/v3.7.7:d7c567b08f, Mar 10 2020, 10:41:24) [MSC v.1900 64 bit (AMD64)]
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.069652 System info dump - END
=MainThread   __main__             DEBUG    2024-10-07 18:49:36.069652 ***************************************************************************
```

Every time ALS tries to save an image, ERROR messages like the following are posted in the UI log :

```
2024-10-07 18:54:38.605815 ERROR   : Failed to save image : C:/als_work/stack_image.jpg
2024-10-07 18:54:38.724695 ERROR   : Failed to save image : C:/als_work/web_image.jpg
```

And a full save sequence shows the following in saved detaled logs :
```
=Dummy-4      als.code_utilities   DEBUG    2024-10-07 18:54:38.646781 SignalingQueue.get() called with : (<als.code_utilities.SignalingQueue object at 0x000001B2BC3F6798>,) - {}
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.646781 Controller.on_saver_busy() called with : (<als.logic.Controller object at 0x000001B2C8B9EBC8>,) - {}
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.646781 Controller._notify_model_observers() called with : (<als.logic.Controller object at 0x000001B2C8B9EBC8>,) - {}
=Dummy-4      als.code_utilities   DEBUG    2024-10-07 18:54:38.647825 SignalingQueue.get() returned Image(ID=-9223371920141762120, Color=True, Exp. t=4, Needs Debayer=False, Bayer Pattern=, Width=1304, Height=976, Data shape=(976, 1304, 3), Data type=uint16, Origin=Process result, Destination=C:/als_work/web_image.jpg in 0.000 ms
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.647825 MainWindow.update_display() called with : (<als.ui.windows.MainWindow object at 0x000001B2C8BAB048>, False) - {}
=Dummy-4      als.processing       INFO     2024-10-07 18:54:38.648419 Start save on Process result
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.649020 MainWindow._update_issues_button_visibility() called with : (<als.ui.windows.MainWindow object at 0x000001B2C8BAB048>,) - {}
=Dummy-4      als.io.output        DEBUG    2024-10-07 18:54:38.649020 ImageSaver._handle_item() called with : (<als.io.output.ImageSaver object at 0x000001B2C8B47E58>, Image(ID=-9223371920141762120, Color=True, Exp. t=4, Needs Debayer=False, Bayer Pattern=, Width=1304, Height=976, Data shape=(976, 1304, 3), Data type=uint16, Origin=Process result, Destination=C:/als_work/web_image.jpg) - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.649569 MainWindow._update_issues_button_visibility() returned None in 0.000 ms
=Dummy-4      als.model.base       DEBUG    2024-10-07 18:54:38.649569 Image.clone() called with : (Image(ID=-9223371920141762120, Color=True, Exp. t=4, Needs Debayer=False, Bayer Pattern=, Width=1304, Height=976, Data shape=(976, 1304, 3), Data type=uint16, Origin=Process result, Destination=C:/als_work/web_image.jpg,) - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.649569 MainWindow.update_display() returned None in 1.150 ms
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.649569 Controller._notify_model_observers() returned None in 1.744 ms
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.650586 Controller.on_saver_busy() returned None in 2.788 ms
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.650586 Controller.on_saver_queue_size_changed() called with : (<als.logic.Controller object at 0x000001B2C8B9EBC8>, 0) - {}
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.650586 *SD-Q-SAV* New saver queue size : 0
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.650586 Controller._notify_model_observers() called with : (<als.logic.Controller object at 0x000001B2C8B9EBC8>,) - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.650586 MainWindow.update_display() called with : (<als.ui.windows.MainWindow object at 0x000001B2C8BAB048>, False) - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.650586 MainWindow._update_issues_button_visibility() called with : (<als.ui.windows.MainWindow object at 0x000001B2C8BAB048>,) - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.650586 MainWindow._update_issues_button_visibility() returned None in 0.000 ms
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.652233 MainWindow.update_display() returned None in 0.000 ms
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.652233 Controller._notify_model_observers() returned None in 1.647 ms
=MainThread   als.logic            DEBUG    2024-10-07 18:54:38.652233 Controller.on_saver_queue_size_changed() returned None in 1.647 ms
=Dummy-4      als.model.base       DEBUG    2024-10-07 18:54:38.652233 Image.clone() returned Image(ID=-9223371920143446280, Color=True, Exp. t=4, Needs Debayer=False, Bayer Pattern=, Width=1304, Height=976, Data shape=(976, 1304, 3), Data type=uint16, Origin=Process result, Destination=C:/als_work/web_image.jpg in 2.664 ms
=Dummy-4      als.io.output        DEBUG    2024-10-07 18:54:38.652233 ImageSaver._save_image() called with : (Image(ID=-9223371920143446280, Color=True, Exp. t=4, Needs Debayer=False, Bayer Pattern=, Width=1304, Height=976, Data shape=(976, 1304, 3), Data type=uint16, Origin=Process result, Destination=C:/als_work/web_image.jpg,) - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.653196 MainWindow.on_message() called with : (<als.ui.windows.MainWindow object at 0x000001B2C8BAB048>, '2024-10-07 18:54:38.649020 INFO    : Start save on Process result') - {}
=Dummy-4      als.io.output        DEBUG    2024-10-07 18:54:38.653196 ImageSaver._save_image_as_jpg() called with : (Image(ID=-9223371920143446280, Color=True, Exp. t=4, Needs Debayer=False, Bayer Pattern=, Width=1304, Height=976, Data shape=(976, 1304, 3), Data type=uint16, Origin=Process result, Destination=C:/als_work/web_image.jpg, 'ALZweb_image.jpg') - {}
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.657469 MainWindow.on_message() returned None in 4.273 ms
=Dummy-4      als.io.output        DEBUG    2024-10-07 18:54:38.718097 ImageSaver._save_image_as_jpg() returned (True, '') in 59.617 ms
=Dummy-4      als.io.output        ERROR    2024-10-07 18:54:38.723694 Failed to save image : C:/als_work/web_image.jpg
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.724695 MainWindow.on_message() called with : (<als.ui.windows.MainWindow object at 0x000001B2C8BAB048>, '2024-10-07 18:54:38.724695 ERROR   : Failed to save image : C:/als_work/web_image.jpg') - {}
=Dummy-4      als.io.output        DEBUG    2024-10-07 18:54:38.724695 ImageSaver._save_image() returned None in 71.499 ms
=MainThread   als.ui.windows       DEBUG    2024-10-07 18:54:38.729137 MainWindow.on_message() returned None in 4.442 ms
=Dummy-4      als.io.output        DEBUG    2024-10-07 18:54:38.729664 ImageSaver._handle_item() returned None in 80.095 ms
=Dummy-4      als.processing       INFO     2024-10-07 18:54:38.729664 End save on Process result in 80.644 ms
```

**Comment by deufrai** [comment]
fix to be tested with custom build at https://als-app.org/assets/support/issue_195/als.exe

**Comment by deufrai** [comment]
Fixed with 9f28619
